.travis.yml

```diff
@@ -3,22 +3,27 @@ language: php
 matrix:
   include:
     - php: 5.3
+      env: COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.4
     - php: 5.5
     - php: 5.6
       env: SCRUTINIZER_REPORT=true
     - php: 7.0
+      env: SYMFONY_VERSION='2.8.*'
+    - php: 7.0
+      env: SYMFONY_VERSION='3.0.*'
 
 cache:
   directories:
     - vendor
-    - $HOME/.composer/cache
+    - $HOME/.composer/cache/files
 
 before_install:
   - composer self-update
+  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --dev --no-update symfony/symfony=$SYMFONY_VERSION; fi
 
 install:
-  - composer install --prefer-dist
+  - composer update $COMPOSER_FLAGS --prefer-dist --optimize-autoloader
```

Comment:
We should never run composer update, since that's to be done for developers on their local machines only afaik. I don't see what this is gaining us from the previous setup where we just did composer install?

Might be something that I just don't get, please explain :)